### PR TITLE
chore(fedimint-api-client): cleanup weird APIs

### PR DESCRIPTION
**Pull request by dpc**
Cleanups after #4956

### CODE REVIEW

File: fedimint-api-client/src/api.rs [hunk 1]
```diff
@@ -540,8 +540,12 @@ impl DynGlobalApi {
         .into()
     }
 
-    pub fn from_invite_code(invite_code: &[InviteCode]) -> Self {
-        GlobalFederationApiWithCache::new(WsFederationApi::from_invite_code(invite_code)).into()
+    pub fn from_invite_code(invite_code: &InviteCode) -> Self {
+        // PeerIds are used only for informational purposes, but just in case, make a
+        // big number so it stands out
+        let peer_id = PeerId::from(1025);
```
Comment (elsirion) [inline fedimint-api-client/src/api.rs:546]: 1. The invite code contains the `PeerId` belonging to its URL

https://github.com/fedimint/fedimint/blob/f604962d884a023b5d7271a0faf0a10cee6b4319/fedimint-core/src/invite_code.rs#L98-L100

2. Invite codes can even contain multiple URLs now

https://github.com/fedimint/fedimint/blob/f604962d884a023b5d7271a0faf0a10cee6b4319/fedimint-core/src/invite_code.rs#L110-L119

**Review by elsirion — DISMISSED** [review]

File: fedimint-api-client/src/api.rs [hunk 2]
```diff
@@ -567,8 +568,13 @@ impl DynGlobalApi {
         .into()
     }
 
-    pub fn from_invite_code(invite_code: &[InviteCode]) -> Self {
-        GlobalFederationApiWithCache::new(WsFederationApi::from_invite_code(invite_code)).into()
+    pub fn from_invite_code(invite_code: &InviteCode) -> Self {
+        // PeerIds are used only for informational purposes, but just in case, make a
+        // big number so it stands out
```
Comment (maan2003) [inline fedimint-api-client/src/api.rs:573]: outdated

Comment (maan2003) [reply fedimint-api-client/src/api.rs]: fixed

**Review by oleonardolima — APPROVED** [review]
ACK

**Review by maan2003 — APPROVED** [review]

**Pull request merged**